solr/core/src/java/org/apache/solr/core/snapshots/SolrSnapshotsTool.java

```diff
@@ -199,7 +200,7 @@ public void describeSnapshot(String collectionName, String snapshotName) {
           + " is not found for collection " + collectionName);
     }
 
-    DocCollection collectionState = solrClient.getZkStateReader().getClusterState().getCollection(collectionName);
+    DocCollection collectionState = ZkStateReader.from(solrClient).getClusterState().getCollection(collectionName);
```

Comment:
In most/all cases `ZkStateReader.from(solrClient).getClusterState()` isn't needed -- just call solrClient.getClusterStateProvider().getClusterState() (or something like that).  I won't repeat this comment; you can find them.